osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyCursorParticles.cs

```diff
@@ -67,12 +74,26 @@ private void load(ISkinSource skin)
                     Colour = starBreakAdditive,
                     Direction = SpewDirection.None,
                 },
+                relaxHitSpewer = new LegacyRelaxCursorParticleSpewer(texture, 60)
+                {
+                    Anchor = Anchor.Centre,
+                    Origin = Anchor.Centre,
+                    Colour = starBreakAdditive,
+                    Direction = SpewDirection.RelaxHit,
+                }
             };
 
             if (player != null)
                 ((IBindable<bool>)breakSpewer.Active).BindTo(player.IsBreakTime);
+
+            relaxing = isRelaxing();
+
+            if (relaxing)
+                kiaiSpewer.Active.BindValueChanged(_ => { });
```

Comment:
What is even happening here? Why does this conditionally register an _empty_ `BindValueChanged()` callback?

Reply:
sorry for that, i forgot to clean it after debugging.

Reply:
I tried `player.Mods.BindValueChanged(v => relaxing = v.NewValue.OfType<OsuModRelax>().Any());` and found this callback was never called. Maybe that's the reason.